src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -253,49 +273,69 @@ export const ProgressBar = (): JSX.Element => {
                     </CardContent>
                 </Card>
                 <List data-cy={'list-content'} disablePadding component="nav" className={classes.placementOfList}>
-                    {fileUploadList.map(
-                        (item, i): JSX.Element => (
-                            <div key={`itemKey${item.id}`} className={classes.infoList}>
-                                <Snackbar
-                                    classes={{ root: classes.snackbar }}
-                                    open={true}
-                                    autoHideDuration={item.progress === 100 ? 3000 : null}
-                                    onClose={(e, reason): void => handleRequestClose(e, reason, item.id)}
+                    <TransitionGroup>
+                        {fileUploadList.map(
+                            (item, i): JSX.Element => (
+                                <CSSTransition
+                                    key={`transitionKey${item.id}`}
+                                    timeout={500}
+                                    className={classes.snackbarItem}
+                                    classNames={{
+                                        enter: classes.snackbarEnter,
+                                        enterActive: classes.snackbarEnterActive,
+                                        exit: classes.snackbar,
+                                        exitActive: classes.snackbarActive,
+                                    }}
```

Comment:
This seems to be rather redundant since there is already a transition element built in to the snackbar component. I thought we were going to try to use the `TransitionProps` property on the snackbar to handle the animating in and out?

Reply:
Fixed. Please check.